### DISCUSSION TITLE
Add custom storage locations

### DISCUSSION THREAD

**Discussion by BuTingBa** [immich-app/immich#11612] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to be able to set a location for each user to store the photo source file.
This makes it easy for users to manage their own files, not just albums

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I'm not sure what you're looking for, but between the storage label and being able to set custom docker mounts, everything should already be possible.
